python/lsst/dax/imgserv/vo/imageSODA.py

```diff
@@ -70,7 +72,8 @@ def do_sync(self, params: dict) -> object:
                                       "Request")
 
     def do_async(self, params: dict) -> object:
```

Comment:
is it an object, or a string?  Using strong typing and then object isn't particularly helpful, it seems like it should be str?

Reply:
Have to revisit how we want to provide type 'hints' for return values.  In general, I don't find this useful at all, given the prevailing practice in Python. Maybe it's better to leave it out.